templates/Wpf/Pages/XamlIsland/Param_ProjectName/Controls/wts.ItemNameControl.xaml.cs

```diff
@@ -51,7 +48,7 @@ private void OnChildChanged(object sender, EventArgs e)
             {
                 _universalControl = xamlIsland;
                 ApplyColors();
-                _universalControl.SetBinding(wts.ItemNameControlUniversal.TextProperty, new WUXD.Binding() { Path = new WUX.PropertyPath(nameof(Text)), Mode = WUXD.BindingMode.TwoWay });
+                _universalControl.SetBinding(XAMLIslandControlUniversal.TextProperty, new Windows.UI.Xaml.Data.Binding() { Path = new Windows.UI.Xaml.PropertyPath(nameof(Text)), Mode = Windows.UI.Xaml.Data.BindingMode.TwoWay });
```

Comment:
Think this should be wts.ItemNameControlUniversal. instead of XAMLIslandControlUniversal.